Firestore/Source/CMakeLists.txt

```diff
@@ -90,4 +91,83 @@ if(APPLE)
     PUBLIC
     ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public
   )
-endif()
+
+  objc_framework(
+    FirebaseFirestore
+    HEADERS
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FirebaseFirestore.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRCollectionReference.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRDocumentChange.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRDocumentReference.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRDocumentSnapshot.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRFieldPath.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRFieldValue.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRFirestore.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRFirestoreErrors.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRFirestoreSettings.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRFirestoreSource.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRGeoPoint.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRListenerRegistration.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRQuery.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRQuerySnapshot.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRSnapshotMetadata.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRTimestamp.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRTransaction.h
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FIRWriteBatch.h
+    SOURCES
+      API/FIRCollectionReference+Internal.h
```

Comment:
Is there any way to get this to depend on any of the existing libraries that we've already defined?